src/nupic/encoders/ScalarEncoder.cpp

```diff
@@ -124,7 +124,7 @@ void ScalarEncoder::encode(Real64 input, SDR &output)
 {
   // Check inputs
   NTA_CHECK( output.size == size );
-  if( isnan(input) ) {
+  if( std::isnan(input) ) {
```

Comment:
nit, this is cpp, fine to include namespace std

Reply:
haha I think I got carried away copy-pasting `std::` all over the place...
I'm going to leave this here & merge since it works as it is.